manifests/vrrp/track_process.pp

```diff
@@ -0,0 +1,39 @@
+# == Define: keepalived::vrrp::track_process
+#
+# === Parameters:
+#
+# $ensure::   Default: present
+#
+# $proc_name:: process name to track
+#              Default: undef (error)
+#
+# $weight::   The weight that should add to the instance.
+#             Default: undef (see keepalived.conf(5) for default)
+#
+# $quorum::   Number of processes to expect running
+#             Default: 1
+#
+# $delay::    Time to delay after process quorum lost before
+#             considering process failed (in fractions of second)
+#             Default: undef
+#
+# $fullcommand::  Match entire process cmdline
+#             Default: undef
+#
+define keepalived::vrrp::track_process (
+  $proc_name   = undef,
+  $weight      = undef,
+  $quorum      = '1',
+  $delay       = undef,
+  $fullcommand = undef
+) {
+  if ! $proc_name {
```

Comment:
Could you make the parameter non optional instead (remove its default)?  I'm fixing this for script.pp in https://github.com/voxpupuli/puppet-keepalived/pull/163

The odd construct was left over from https://github.com/voxpupuli/puppet-keepalived/commit/3a529bc7310782f55b03774d1c4e3a044d86549a#diff-5b0dbec169818e08fe4324e5386753c6L10